### ISSUE TITLE
New languages

### ISSUE THREAD

**Issue by kaalleen** [inkstitch/inkstitch#527]
When will a newly supported languages show up on the release page?

We got translations into Turkish and Ukrainian (about 10%). It would be nice if we could offer builds for these languages now. I approved their translations just recently. I would have expected to see at least the Turkish language build in the troubleshoot PR (Ukrainian was approved a bit later).

**Comment by kaalleen** [comment]
Do we need a certain percentage of translated strings?
I just saw that the 11% translations into Portuguese also never made it into a release.
If so, that's fine. But we should inform translators about it, because it might be discouraging otherwise.

**Comment by lexelby** [comment]
I had to dig around a bit to remember how this worked.  It looks like I was really "clever" when I implemented this.  Perhaps too clever? ;)

Check this out: https://github.com/inkstitch/inkstitch/blob/master/lib/inx/utils.py#L55

Once a translator enters a translation for the phrase "Generate INX files", that's what causes the build system to build that language.  The `L10N` comment _should_ appear in CrowdIn to explain how this works.
